### PR TITLE
Enable and refactor binance fetcher

### PR DESCRIPTION
**Pull request by gagiuntoli**
## Context

Feeds.yaml example

```yaml
STMX-USDT:
  discrepancy: 1.0
  precision: 8
  heartbeat: 86400
  trigger: 0.5
  interval: 60
  chains: [rootstock]
  inputs:
    - fetcher:
        name: Binance
        params:
          symbol: STMXUSDT

BTC-USDT:
  discrepancy: 1.0
  precision: 8
  heartbeat: 86400
  trigger: 0.5
  interval: 60
  chains: [rootstock]
  inputs:
    - fetcher:
        name: Binance
        params:
          symbol: BTCUSDT

USDT-BTC:
  discrepancy: 1.0
  precision: 8
  heartbeat: 86400
  trigger: 0.5
  interval: 60
  chains: [rootstock]
  inputs:
    - fetcher:
        name: Binance
        params:
          symbol: BTCUSDC
          inverse: true
```

## To-do list

- [ ] Added tests
- [ ] Tested on dev/sbx
- [ ] Changelog was updated with current changes
- [ ] Documentation or guides were updated (slab, readme)
- [ ] Version was updated on `package.json` (releases/hotfixes)

## Checklist before merge

- [ ] **there are no errors in logs in any workers**
- [ ] new blocks are being discovered and marked as finalized
- [ ] foreign blocks are being dispatched
- [ ] squash all commits before merging

### CODE REVIEW

File: src/services/feedProcessors/BinanceMultiProcessor.ts [hunk 1]
```diff
@@ -48,35 +48,20 @@ export default class BinanceMultiProcessor {
...
-      const {id, currency} = fetcher.params as InputParams;
-      inputsIndexMap[`${id}:${currency}`] = index;
-    });
-
-    const result: number[] = [];
-    result.length = feedFetchers.length;
-
-    values.forEach(({id, currency, value}) => {
-      const index = inputsIndexMap[`${id}:${currency}`];
+      if (fetcher.name == FetcherName.BINANCE) {
```
Comment (DZariusz) [inline src/services/feedProcessors/BinanceMultiProcessor.ts:58]: I would keep the way we do in all other places
```suggestion
      if (fetcher.name != FetcherName.BINANCE) return;
```

File: test/services/fetchers/binancePriceMultiFetcher.test.ts [hunk 2]
```diff
@@ -15,12 +14,9 @@ describe.only('BinancePriceMultiFetcher', () => {
   let settings: Settings;
   let binancePriceMultiFetcher: BinancePriceMultiFetcher;
 
-  const params: InputParams[] = [
-    {id: 'BTC', currency: 'USDT'},
-    {id: 'ETH', currency: 'USDT'},
-  ];
+  const params: InputParams[] = [{symbol: 'BTCUSDT'}, {symbol: 'ETHUSDT'}];
```
Comment (DZariusz) [inline test/services/fetchers/binancePriceMultiFetcher.test.ts:17]: will `USDTBTC` work? if not we need to add option to reverse the price so we can have it both ways.

**Review by DZariusz — APPROVED** [review]

File: test/services/fetchers/binancePriceMultiFetcher.test.ts (hunk 2 above)
Comment (gagiuntoli) [reply test/services/fetchers/binancePriceMultiFetcher.test.ts:17]: Hello Dariousz, the price USDTBTC is not in available in the Binance API.

Would you like that we check in the case we don't find it to try to fetch `BTCUSDT` and then we invert it?

File: src/services/feedProcessors/BinanceMultiProcessor.ts (hunk 1 above)
Comment (gagiuntoli) [reply src/services/feedProcessors/BinanceMultiProcessor.ts:58]: done

File: test/services/fetchers/binancePriceMultiFetcher.test.ts (hunk 2 above)
Comment (DZariusz) [reply test/services/fetchers/binancePriceMultiFetcher.test.ts:17]: yes, if not avilable, just revert ir

**Review by DZariusz — APPROVED** [review]

**Pull request merged**